tools/tests/fixtures/calendar-grid-temporal-groups-known-bad.html

```
@@ -0,0 +1,17 @@
+<table aria-label="Known-bad Calendar Grid header">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)